snapshots.go

```diff
@@ -137,3 +137,19 @@ func (RevertSnapshot) Response() interface{} {
 func (RevertSnapshot) AsyncResponse() interface{} {
 	return new(BooleanResponse)
 }
+
+// ExportSnapshot exports an instant snapshot of a volume to SOS
+type ExportSnapshot struct {
+	ID *UUID `json:"id"`
+	_  bool  `name:"exportSnapshot" description:"Exports an instant snapshot of a volume."`
+}
+
+// ExportSnapshotResponse represents the presigned URL to access the exported snapshot
```

Comment:
There will be more information returned besides the pre-signed URL at some point (e.g. the file checksum).

```suggestion
// ExportSnapshotResponse represents the response of a snapshot export operation.
```